### ISSUE TITLE
Do you have a plan to provide "tabs" in your software?

### ISSUE THREAD

**Issue by MingyuKim87** [junghyun3/my-dictionary-v2#2]
I love this SW, my-dictionary.
It is very useful and giving me more comfortable to look for some information.
However, when I browse multiple website such as Papago and ludwig, I should experience refreshing website during transit from a site to other site. Especially, it took 1~2 seconds, it bothered me. 

Second, could you make shortcut of my-dictionary when I activate other SW?
It means that I would like to use shortcuts of my-dictionary as a first priority than other SW. 
For example, when I browse a website using safari, shortcut of my-dictionary like "command-1" dose not work. 
In order to use this shortcut, I have to place on my cursor at my-dictionary, and then it works. 

In conclusion, I would like to know what plan you have to improve this SW.

Lastly, I really appreciate your effort and commitment.
